packages/moby-runc/deb.mk

```
@@ -2,7 +2,7 @@ deb: runc man/man8
 
 runc:
 	cd src && \
-	$(MAKE) runc BUILDTAGS='seccomp' VERSION="${VERSION}-${REVISION}"
+	$(MAKE) runc BUILDTAGS='seccomp urfave_cli_no_docs' VERSION="${VERSION}-${REVISION}" COMMIT=${COMMIT}
```

```suggestion
	$(MAKE) runc BUILDTAGS='seccomp urfave_cli_no_docs' VERSION="${VERSION}-${REVISION}" COMMIT="${COMMIT}"
```

`rpm.mk` has quotes around `${COMMIT}`, so this should too.

---

Updated